xwalk_android_tests.gypi

```diff
@@ -147,6 +147,7 @@
           '<(PRODUCT_DIR)/xwalk_xwview_test/assets/create_window_2.html',
           '<(PRODUCT_DIR)/xwalk_xwview_test/assets/console_message.html',
           '<(PRODUCT_DIR)/xwalk_xwview_test/assets/echo_java.html',
+          '<(PRODUCT_DIR)/xwalk_xwview_test/assets/echo_binary_java.html',
```

Comment:
@huningxin One trial comment on  the order. echo_binary_java should before echo_java. Ditto for rest.